src/clarity-angular/nav/_nav.clarity.scss

```diff
@@ -49,8 +49,7 @@ $clr-active-nav-link-shadow: 0 -3px 0 $action-blues-dark-midtone inset;
             }
 
             &:hover,
-            &.active,
-            &:focus {
```

Comment:
@adityarb88: yes, with the change above the browser default focus state will show but only if the user tabbed to it and hit the back button (after fix screenshot below). If the user clicks on it and hits the back button then it won't highlight it at all.

Let me know if my explanation is confusing.

Before fix:
<img width="481" alt="screen shot 2017-01-25 at 12 01 24 pm" src="https://cloud.githubusercontent.com/assets/1827742/22306999/382b085a-e2f6-11e6-929e-32787260b1a9.png">

After fix:
<img width="514" alt="screen shot 2017-01-25 at 12 00 40 pm" src="https://cloud.githubusercontent.com/assets/1827742/22307006/3dbf51d6-e2f6-11e6-8640-fe2801e8869a.png">